### PR TITLE
[demo] Replace references to feature flag service with flagd

### PR DESCRIPTION
**Pull request by austinlparker**
docs updates for v1.9, ff service deprecation in favor of flagd.

### CODE REVIEW

**Comment by austinlparker** [comment]
This is waiting for https://github.com/open-telemetry/opentelemetry-demo/pull/1388 to merge before it can be reviewed

File: content/en/docs/demo/feature-flags.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
 | Feature Flag            | Service(s)      | Description                                                                                              |
 | ----------------------- | --------------- | -------------------------------------------------------------------------------------------------------- |
 | `adServiceFailure`      | Ad Service      | Generate an error for `GetAds` 1/10th of the time                                                        |
 | `cartServiceFailure`    | Cart Service    | Generate an error for `EmptyCart` 1/10th of the time                                                     |
 | `productCatalogFailure` | Product Catalog | Generate an error for `GetProduct` requests with product id: `OLJCESPC7Z`                                |
 | `recommendationCache`   | Recommendation  | Create a memory leak due to an exponentially growing cache. 1.4x growth, 50% of requests trigger growth. |
+
+## Feature Flag Architecture
+
+Please see the [flagd documentation](https://github.com/open-feature/flagd) for
```
Comment (beeme1mr) [inline content/en/docs/demo/feature-flags.md:22]:
```suggestion
Please see the [flagd documentation](https://flagd.dev) for
```

File: content/en/docs/demo/telemetry-features/_index.md [hunk 2]
```diff
@@ -51,3 +52,7 @@ aliases: [demo_features, features]
   flag service.
 - **[Locust](https://locust.io)**: a background job that creates realistic usage
   patterns on the website using a synthetic load generator.
+- **[OpenFeature](https://openfeature.dev)**: a feature flagging API and SDK
+  that allows for the enabling and disabling of features in the application.
+- **[flagd](https://github.com/open-feature/flagd)**: a feature flagging daemon
```
Comment (beeme1mr) [inline content/en/docs/demo/telemetry-features/_index.md:57]:
```suggestion
- **[flagd](https://flagd.dev)**: a feature flagging daemon
```

File: content/en/docs/demo/requirements/architecture.md [hunk 3]
```diff
@@ -68,16 +68,8 @@ application written in Swift.
...
-browser-based client/admin interface and a backend service or services. The role
-of the client is to allow an operator to visualize the available feature flags
-and toggle their state. The server should provide a catalog of feature flags
-that main application services can register with and interrogate for their
-current status and targeting rules.
-
-The feature flag component should be implemented as an Erlang+Elixir/Phoenix
-service. The catalog of feature flags should be stored in a Database.
+Feature flagging is a crucial part of cloud-native application development. The
+demo should use OpenFeature, a CNCF incubating project, to manage feature flags.
```
Comment (julianocosta89) [inline content/en/docs/demo/requirements/architecture.md:72]:
```suggestion
demo uses OpenFeature, a CNCF incubating project, to manage feature flags.
```

**Review by julianocosta89 — APPROVED** [review]
LGTM, just a minor comment.

**Review by svrnm — APPROVED** [review]

File: content/en/docs/demo/feature-flags.md [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
-cSpell:ignore: OLJCESPC7Z
+cSpell:ignore: flagd OLJCESPC7Z
 ---
 
-This demo comes with several feature flags which can control failure conditions
-in specific services. By default the flags are disabled. Using the Feature Flags
-UI <http://localhost:8080/feature> you will be able to control the status of
-these feature flags.
+The demo provides several feature flags that you can use to simulate different
+scenarios. These flags are managed by [`flagd`](flagd.dev), a simple feature flag service
```
Comment (svrnm) [inline content/en/docs/demo/feature-flags.md:8]:
```suggestion
scenarios. These flags are managed by [`flagd`](https://flagd.dev), a simple feature flag service
```

**Comment by svrnm** [comment]
I forgot an https in my suggestion, fixed that.

@austinlparker I have a list of PRs I am going to merge right now, I will handle this one as well.

**Pull request merged**